wordpress-mobile/gutenberg-mobile
# Updating gutenberg ref, fix for crash on pasting formatted text.

### Fixes https://app.circleci.com/jobs/github/wordpress-mobile/gutenberg-mobile/17951/tests

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/18740

### Video of crash:
![crash-on-paste-formatted](https://user-images.githubusercontent.com/1103838/69599963-bea8e300-0fc3-11ea-9578-57990263d967.gif)

### To test:

Note that with the updated gutenberg ref that adds a check for undefined, we are now seeing all Android on device tests passing.

Update release notes:
